apps/notifications/package.json

```diff
@@ -56,7 +56,7 @@
 		"@automattic/wp-babel-makepot": "workspace:^",
 		"html-webpack-plugin": "^5.6.3",
 		"postcss": "^8.5.3",
-		"postcss-custom-properties": "^11.0.0",
+		"postcss-custom-properties": "^12.0.0",
```

Comment:
The update to v12 is an attempt to fix a build errors in the apps. I will move to a separate PR if the fix works well